tsconfig.json

```diff
@@ -13,11 +13,15 @@
     "baseUrl": "./",
     "paths": {
       "*" : ["src/@types/*"]
-    }
+    },
+    "target": "es2018"
   },
   "files": [
     "src/client.ts",
+    "src/api.ts",
+    "src/channel.ts",
     "src/user.ts",
+    "src/team.ts",
```

Comment:
What about the other ts files? Aren't they also needed here? Can we order them alphabetically?

Reply:
NICE - fixed!